devel/npm6/Portfile

```diff
@@ -31,11 +31,13 @@ checksums           sha1    4f39f9337b557a28faed4a771d5c8802d6b4288b \
 
 worksrcdir          "package"
 
-depends_lib         path:bin/node:nodejs8
+depends_lib         path:bin/node:nodejs10
+
+revision            1
```

Comment:
`revision` is usually placed after `version`.

Reply:
OK, change made.

Reply:
@l2dy If you're happy, please merge this.